app/views/layouts/_footer.html.erb

```diff
@@ -1,18 +1,23 @@
 <footer class="crayons-footer print-hidden">
-  <div id="footer-container" class="crayons-layout crayons-layout--limited-m crayons-footer__container">
-    <p class="fs-s crayons-footer__description"><a href="/" aria-label="<%= t("views.main.aria_home", community: community_name) %>" class="crayons-link"><%= t("views.main.footer.desc", community: community_name, desc: Settings::Community.community_description) %></p>
-    <div class="m:-mb-4 crayons-footer__description">
+  <div id="footer-container" class="crayons-footer__container">
+    <p class="fs-s crayons-footer__description">
+      <a href="<%= root_path %>" aria-label="<%= t("views.main.aria_home", community: community_name) %>" class="c-link c-link--branded fw-medium">
+        <%= community_name %>
+      </a>
+      – <%= Settings::Community.community_description %>
+    </p>
```

Comment:
Apparently there was a bug introduced (lack of closing `</a>`) while creating a translation for this piece of text. And the translation itself isn't really necessary here since the entire translation is actually generated through Admin. So it's not really a translation anyway.